cypress/e2e/3-testing/logout.cy.js

```diff
@@ -0,0 +1,5 @@
+describe('noroff logout', () => {
+  it('can log out from application', () => {
+    cy.get('#logout-button').click();
```

Comment:
How can you prove that this worked? 

Clicking the button is not enough to test the functionality. Do you know that the user was logged in originally? Can you log out if you haven't logged in?